src/OrchardCore.Modules/Orchard.Roles/Services/RoleStore.cs

```diff
@@ -67,8 +67,6 @@ public void UpdateRoles(RolesDocument roles)
         #region IRoleStore<Role>
         public async Task<IdentityResult> CreateAsync(Role role, CancellationToken cancellationToken)
         {
-            cancellationToken.ThrowIfCancellationRequested();
```

Comment:
If cancellationToken now is useless then don't we should remove that parameter?
A better alternative for me would be to provide a cancellation token in GetRolesAsync and UpdateRoles methods.